### PR TITLE
include note about GATK2 warning in the log

### PR DESCRIPTION
**Pull request by martenson**
due to manual installation

### CODE REVIEW

**Comment by bgruening** [comment]
:+1:

File: tools/gatk2/readme.rst [hunk 1]
```diff
@@ -59,6 +59,10 @@ Finally, you should fill in additional information about your genomes and
 annotations in the gatk2_picard_index.loc and gatk2_annotations.txt. 
 You can find them in the tool-data/ Galaxy directory.
 
+Note that due to the manual nature of the GATK2 installation you will be getting the following warnings in the Galaxy log:
+```Failed to resolve dependency on 'gatk2', ignoring```. This is because the <requirement type="package">gatk2</requirement> is specified but
```
Comment (nsoranzo) [inline tools/gatk2/readme.rst]: I don't think that quoting with \``` is appropriate for rst files.

**Comment by nsoranzo** [comment]
This should go just after the paragraph

```
Optionally, you may also want to add some commands to be executed before GATK (e.g. to load modules) to the file:

<tool_dependency_dir>/gatk2/default/env.sh
```

and maybe rephrased, since there would be no warning in the logs if this env.sh file is created.

**Comment by nsoranzo** [comment]
Thanks @martenson !
